packages/firestore/test/util/test_platform.ts

```diff
@@ -270,6 +270,10 @@ export class TestPlatform implements Platform {
   randomBytes(nBytes: number): Uint8Array {
     return this.basePlatform.randomBytes(nBytes);
   }
+
+  toByteStreamReader(source: unknown): ByteStreamReader {
+    return this.basePlatform.toByteStreamReader(source);
```

Comment:
I suspect that if you move your Node-specifc `genericBundleReadingTests` logic here, then you can use the TestPlatform throughout and won't have to force override it.

Reply:
See above. It now works for both platforms without hacking.